templates/base/cluster-with-kcp.yaml

```diff
@@ -59,9 +59,15 @@ spec:
           - localhost
           - 127.0.0.1
           - 0.0.0.0
+        extraArgs:
+          tls-cipher-suites: "${TLS_CIPHER_SUITES=TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256}"
```

Comment:
where did we get this recommended list from? why so many? can we pick the most secure one for each k8s version?

Reply:
this is the commonly accepted as secure
same used by Openshift

Reply:
https://github.com/kubernetes/kubernetes/issues/83669